src/presets/netlify/utils.ts

```diff
@@ -92,6 +92,13 @@ export async function writeHeaders(nitro: Nitro) {
   await fsp.writeFile(headersPath, contents);
 }
 
+export function getStaticPaths(nitro: Nitro): string[] {
```

Comment:
This utility can accept `publicAssets` directly. This way we can write simple unit tests for it without all new complexity being added (both fixture and tests and be simplified)